app/models/concerns/course/levels_concern.rb

```diff
@@ -25,4 +25,19 @@ def level_for(experience_points)
   def default_level?
     any?(&:default_level?)
   end
+
+  # Delete and create Course::Level objects so they match new given thresholds.
+  #
+  # @param [Array<Integer>] new_thresholds Array of the new experience point thresholds.
+  # @return [Array<Course::Level>] Level objects with the new thresholds.
+  def mass_update_levels(new_thresholds)
```

Comment:
What if 0 wasn't passed into the array? By right this should be ignored, as it is an implementation quirk of how we implement levels now.

Reply:
This case is handled now.